src/Tgstation.Server.Host/Core/SetupWizard.cs

```diff
@@ -166,8 +167,18 @@ async Task<DatabaseConfiguration> ConfigureDatabase(CancellationToken cancellati
 				await console.WriteAsync(null, true, cancellationToken).ConfigureAwait(false);
 				await console.WriteAsync("Enter the server's address and port (blank for local): ", false, cancellationToken).ConfigureAwait(false);
 				var serverAddress = await console.ReadLineAsync(false, cancellationToken).ConfigureAwait(false);
+				var serverPort = 3306U;
```

Comment:
Can we make this a `uint?` instead to avoid setting the default (which is done under the hood if it isn't present.